manala.composer/defaults/main.yml

```diff
@@ -8,6 +8,7 @@ manala_composer_install_packages_default:
   - unzip
 
 # Bin
+manala_composer_version: ~
```

Comment:
Maybe in a dedicated part of the file (Something like `# Version` instead of `# Bin`)